### ISSUE TITLE
AutoLISP Project Manager or Workspace?

### ISSUE THREAD

**Issue by rtogores** [Autodesk-AutoCAD/AutoLispExt#9]
The AutoLISP Project Manager introduced with 1.3.0 Introduces nothing new. What it does was already available using multi-root Workspaces. In fact managing Workspaces is more complete as they can be closed from the Files menu, something lacking in the Project Manager.  In a Workspace you can include and manage DCL or other resource files, while in the project you can only include LSP files.
The Search feature in the Project Manager does the same as the Edit > Find in files menu option already in VS Code. I believe adding these useless stuff to the extension is confusing and will not help in attracting users to this new environment.

What is really needed is to add to MAKELISPAPP the things that were available in the VLISP Project window, such as:
*    The option to compile files to FAS instead of VLX if no DCL or other resources are needed.
*    Choosing compiler options such as the Internal link mode, so the compiler tries to resolve all explicit function calls by referencing the function's definition in memory.
*    Selecting Localize variables, so the compiler would also remove all local symbol names.
*    Selecting Safe optimize.
*    Selecting Full reports as the Message Mode so we have all the information about the compilation process including errors, warnings and compiler statistics.

A desirable feature would also be to simultaneously load all the files in our Workspace (or Project, if you prefer to call it that way) for debugging.

**Comment by szilvaa-adsk** [comment]
> What it does was already available using multi-root Workspaces. 

Yes, there's some overlap but there's an important difference: "projects" can pick files from a folder while workspace always includes all files from a folder. We are trying to ease migration for people who come from the old VLIDE hence this feature.

> What is really needed is to add to MAKELISPAPP...

Agreed. Another user requested something similar here here: #8. 

> A desirable feature would also be to simultaneously load all the files in our Workspace (or Project, if you prefer to call it that way) for debugging.

Agreed.
